recipes/r-fillpattern/meta.yaml

```
@@ -0,0 +1,78 @@
+{% set version = '1.0.1' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fillpattern
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fillpattern_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fillpattern/fillpattern_{{ version }}.tar.gz
+  sha256: 1c5c0a4a2651dfbeb919a619c3bebd8ac3ec3186f0607d7f7a8aeeefc7cada77
+
+build:
+  skip: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-ggplot2 >=3.5
+  run:
+    - r-base
+    - r-ggplot2 >=3.5
+
+test:
+  commands:
+    - $R -e "library('fillpattern')"           # [not win]
+    - "\"%R%\" -e \"library('fillpattern')\""  # [win]
+
+about:
+  home: https://cmmr.github.io/fillpattern/, https://github.com/cmmr/fillpattern
+  license: MIT
+  summary: Adds distinctive yet unobtrusive geometric patterns where solid color fills are normally
+    used. Patterned figures look just as professional when viewed by colorblind readers
+    or when printed in black and white. The dozen included patterns can be customized
+    in terms of scale, rotation, color, fill, line type, and line width. Compatible
+    with the 'ggplot2' package as well as 'grid' graphics.
+  license_family: MIT
+  license_file:
+    - '{{ environ["PREFIX"] }}/lib/R/share/licenses/MIT'
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - dansmith01
```

(optional) We strongly recommend that the Conda Forge R team be listed as a maintainer for all R package recipes. This helps us provide rapid turnaround times for R version releases.

Please disregard any linter complaints about this.

```suggestion
    - conda-forge/r
    - dansmith01
```